### PR TITLE
Component structure

### PR DESCRIPTION
**Pull request by momo1113**
* Set up the components' folders and sub folders
* Render pure information on the page without styling

### CODE REVIEW

File: ProductDetails/src/components/ProductInfo.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+const ProductInfo = ({ product }) => {
```
Comment (jincheng411) [inline ProductDetails/src/components/ProductInfo.jsx:4]: use props instead of {product} so that others can understand the code easier

**Review by jincheng411 — APPROVED** [review]
Looks great! 
One minor issue was for better naming convention when using functional component the input parameter use 'props'  instead of different objects everytime

File: ProductDetails/test/App.test.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 // Link.react.test.js
 import React from 'react';
 import { shallow, mount, render } from 'enzyme';
-import App from './App.js';
-import SubApp from './SubApp.js'
+import App from '../src/components/Products.js';
```
Comment (laenalea) [inline ProductDetails/test/App.test.js:4]: Jest is throwing an error. I think it wants .jsx, not .js.

-- On that note, like with Jin, are we ignoring incorporating unit tests for now?

File: ProductDetails/src/components/Products.jsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    const id = 14807;
+    axios.get(`/products/${id}`)
+      .then((response) => {
+        const { data } = response;
+        const {
+          category, default_price, description, features, name, slogan,
+        } = data;
+        // eslint-disable-next-line object-curly-newline
+        // eslint-disable-next-line max-len
+        this.setState({ product: { category, default_price, description, features, name, slogan } });
```
Comment (laenalea) [inline ProductDetails/src/components/Products.jsx:38]: ESLint is insane. They want this: 
```suggestion
                this.setState({
          product: {
            category, default_price, description, features, name, slogan,
          },
        });
      })
```

**Review by laenalea — APPROVED** [review]

**Review by gusfel — APPROVED** [review]

**Pull request merged**